engine/apps/alerts/migrations/0071_migrate_labels.py

```diff
@@ -0,0 +1,59 @@
+# Generated by Django 4.2.15 on 2024-11-12 09:33
+import logging
+
+from django.db import migrations
+import django_migration_linter as linter
+
+logger = logging.getLogger(__name__)
+
+
+def migrate_static_labels(apps, schema_editor):
+    AlertReceiveChannelAssociatedLabel = apps.get_model("labels", "AlertReceiveChannelAssociatedLabel")
+    AlertReceiveChannel = apps.get_model("alerts", "AlertReceiveChannel")
+
+    logging.info("Start migrating alert group static labels to integration labels")
+
+    labels_associations_to_create = []
+    alert_receive_channels_to_update = []
+
+    alert_receive_channels = AlertReceiveChannel.objects.filter(alert_group_labels_custom__isnull=False)
+    logging.info(f"Found {alert_receive_channels.count()} integrations with custom alert groups labels")
+    for alert_receive_channel in alert_receive_channels:
```

Comment:
do you have the rough number of how many integrations we have in production? (just curious if it's more like 50,000 or 50 million; there's probably not _that_ many, but just want to double check 🙂)

Reply:
ah okay, in that case this should be very performant, thanks for double checking 🙂